evalml/pipelines/components/estimators/regressors/varmax_regressor.py

```diff
@@ -61,7 +61,7 @@ def __init__(
         trend: Optional[str] = "c",
         random_seed: Union[int, float] = 0,
         maxiter: int = 10,
-        use_covariates: bool = True,
+        use_covariates: bool = False,
```

Comment:
Flipped this default for the sake of speed. My test example did not train within a 5 minute window when `use_covariates` was True, but it ran in <10 seconds when `use_covariates` was False.

Reply:
I think it would be nice to see performance tests for `use_covariates` turned on or off. Could possibly turn it off only for tests if performance is greatly improved with covariates!